docs/docs/pods.md

```diff
@@ -0,0 +1,560 @@
+---
+title: Pods
+---
+
+# Pods
+
+Marathon supports the creation and management of pods. Pods enable you to share storage, networking, and other resources among a group of applications on a single agent, address them as one group rather than as separate applications, and manage health as a unit. Pods allow quick, convenient coordination between applications that need to work together, for instance a primary service and a related analytics service or log scraper. Pods are particularly useful for transitioning legacy applications to a microservices-based architecture.
+
+Currently, Marathon pods can only be created and administered via the `/v2/pods/` endpoint of the REST API, not via the web interface.
+
+# Features
+
+- Co-located containers.
+- Pod-level resource isolation.
+- Pod-level sandbox and ephemeral volumes.
+- Pod-level health checks.
+
+# Quick Start
+
+1. Run the following REST call, substituting your IP and port for `<ip>` and `<port>`:
+
+    ```bash
+    $ curl -X POST -H "Content-type: application/json" -d@/dev/stdin http://<ip>:<port>/v2/pods <<EOF
+    > {
+    >   "id": "/simplepod",
+    >   "scaling": { "kind": "fixed", "instances": 1 },
+    >   "containers": [
+    >     {
+    >       "name": "sleep1",
+    >       "exec": { "command": { "shell": "sleep 1000" } },
+    >       "resources": { "cpus": 0.1, "mem": 32 }
+    >     }
+    >   ],
+    >   "networks": [ {"mode": "host"} ]
+    > }
+    > EOF
+    ```
+
+    **Note:** The pod ID (the `id` parameter in the pod specification above) is used for all interaction with the pod once it is created.
+
+1. Verify the status of your new pod:
+
+    ```bash
+    curl GET http://<ip>:<port>/v2/pods/simplepod::status
+    ```
+
+1. Delete your pod:
+
+    ```bash
+    curl -X DELETE http://<ip>:<port>/v2/pods/simplepod
+    ```
+
+# Technical Overview
+
+A pod is a special kind of Mesos task group, and the tasks or containers in the pod are the group members.* A pod instance’s containers are launched together, atomically, via the Mesos LAUNCH_GROUP call. Containers in pods share networking namespace and ephemeral volumes.
```

Comment:
Add a link to [Mesos LAUNCH_GROUP](https://github.com/apache/mesos/blob/cfeabec58fb2a87076f0a2cf4d46cdd02510bce4/docs/executor-http-api.md#launch_group)